Composer/packages/types/src/indexers.ts

```diff
@@ -116,6 +116,7 @@ export type LuFile = {
   resource: LuParseResource;
   imports: { id: string; path: string; description: string }[];
   published?: boolean;
+  rawData: boolean;
```

Comment:
@lei9444 can you tell me what rawData represents, the naming certainly does not reflect its purpose.

Reply:
I agree, it looks like it should be some data, but its a boolean.

Reply:
Sorry, this is used to check if the file is parsed, I have changed it to isContentUnparsed

Reply:
How about isContentParsed instead?